### PR TITLE
Update reserve_transfer documentation

### PR DESCRIPTION
**Pull request by 0x7CFE**
Reflect recent changes made in the implementation.

### CODE REVIEW

**Comment by 0x7CFE** [comment]
https://github.com/AstarNetwork/astar-frame/pull/108 is merged, now it's good time to review and update the docs.

File: docs/xcm/building-with-xcm/xc-reserve-transfer.md [hunk 1]
```diff
@@ -48,33 +48,25 @@ This functionality is exposed to EVM smart contracts via precompiles. Interface
...
-    { parents: 1, interior: X1(Parachain(2007)) },
-    { parents: 0, interior: Here }
-]
-
-asset_amounts = [ 333333333, 111111 ]
+assets = [{ parents: 0, interior: Here }]
+asset_amounts = [ 333333333 ]
 ```
 
+Native assets can be used for paying transaction fees as any other assets. In that case `fee_index` should point to the `asset_id` entry containing `0xFF...FE`.
```
Comment (Dinonard) [inline docs/xcm/building-with-xcm/xc-reserve-transfer.md:68]: I believe you meant to say `0x00...00`?

Comment (0x7CFE) [reply docs/xcm/building-with-xcm/xc-reserve-transfer.md:68]: Indeed. This is artifact from previous iteration.

**Review by Dinonard — APPROVED** [review]

**Comment by 0x7CFE** [comment]
@akru could you please review this PR

**Review by akru — APPROVED** [review]

**Pull request merged**